cluster-provision/gocli/opts/nodes/nodes.go

```diff
@@ -3,23 +3,53 @@ package nodes
 import (
 	_ "embed"
 	"fmt"
+	"github.com/Masterminds/semver/v3"
+	"github.com/sirupsen/logrus"
+	"regexp"
 	"runtime"
 
 	"kubevirt.io/kubevirtci/cluster-provision/gocli/pkg/libssh"
 )
 
-//go:embed conf/00-cgroupv2.conf
-var cgroupv2 []byte
+var (
+	//go:embed conf/00-cgroupv2.conf
+	cgroupv2 []byte
+
+	versionRegex *regexp.Regexp
+
+	v1_32 *semver.Version
+)
 
 type nodesProvisioner struct {
+	k8sVersion  string
 	sshClient   libssh.Client
 	singleStack bool
+	version     *semver.Version
+}
+
+func init() {
+	versionRegex = regexp.MustCompile(`.*([0-9]+\.[0-9]+)`)
+	var err error
+	v1_32, err = semver.NewVersion("v1.32")
+	if err != nil {
+		logrus.Fatalf("not a parseable semver contained in %q", "v1.32")
+	}
 }
 
-func NewNodesProvisioner(sc libssh.Client, singleStack bool) *nodesProvisioner {
+func NewNodesProvisioner(k8sVersion string, sc libssh.Client, singleStack bool) *nodesProvisioner {
+	submatches := versionRegex.FindStringSubmatch(k8sVersion)
+	if len(submatches) != 2 {
+		logrus.Fatalf("not a parseable semver contained in %q", k8sVersion)
+	}
```

Comment:
@dhiller - This is breaking several periodic jobs, e.g. `periodic-kubevirt-push-nightly-conformance` and `periodic-hco-push-nightly-build-main`